### PR TITLE
Disable Change Password for any user other than the currently logged in

### PR DESCRIPTION
**Pull request by fjvallarino**
@ezequielc please review

### CODE REVIEW

File: src/templates/user-settings-modal.html [hunk 1]
```diff
@@ -45,12 +45,12 @@ <h2 id="user-settings-label" class="modal-title">
...
       </div>
       <div class="col-xs-6 text-right">
-        <span ng-if="!isAdd">
+        <span ng-if="editingYourself && !isAdd">
           <a href="" class="btn btn-default btn-sm change-password" ng-click="toggleChangePassword()">Change password</a>
         </span>
       </div>
     </div>
-    <div id="passwordForm" class="animated" ng-if="!isAdd" ng-show="isRiseAuthUser && showChangePassword">
+    <div id="passwordForm" class="animated" ng-if="editingYourself && !isAdd" ng-show="isRiseAuthUser && showChangePassword">
```
Comment (ezequielc) [inline src/templates/user-settings-modal.html]: @fjvallarino Do you need `editingYourself` here? If we hide the button above, how could this be displayed? Thanks

Comment (fjvallarino) [reply src/templates/user-settings-modal.html]: @ezequielc I think that actually applies to line 18 too. I added them for the sake of safety, but removing them probably makes sense

Comment (ezequielc) [reply src/templates/user-settings-modal.html]: @fjvallarino I am good with keeping them for now. Thanks

**Comment by ezequielc** [comment]
@fjvallarino 👍

**Pull request merged**